meson.build

```diff
@@ -42,6 +42,11 @@ if with_coverage
     '-fprofile-dir=' + join_paths(get_option('prefix'), get_option('localstatedir'), 'tmp', 'bluechi-coverage'),
     language : 'c')
   add_project_link_arguments('-lgcov', language : 'c')
+
+  # Compiler should not perform any optimization when code coverage is reported, but meson RPM support sets -O2
+  # into CFLAGS, so we need to override here
+  common_cflags += '-Og'
```

Comment:
I don't know the `-0g` flag. Could you point me to documentation?

Reply:
Ah, think I found it: 
https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html#index-Og

Reply:
Yeah, that's the right link